modules/ibi-ib-cli-reference.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/ibi-image-based-install.adoc 
+
+:_content-type: REFERENCE
+[id="ibi-ib-cli-reference_{context}"]
+= ib-cli CLI tool API specifications
+
+The following matrix describes the specifications for the `image-based-install-iso.yaml`. The `ib-cli` CLI tool uses the `image-based-install-iso.yaml` file to create a live installation ISO.
+
+.Required specifications
+[options="header"]
+[cols="2a,1a,4a"]
+|====
+|Specification|Type|Description
+|`seedImage`|`string`|Specifies the seed image to use in the ISO generation process.
+|`seedVersion`|`string`|Specifies the {product-title} release version of the seed image.
+|`installationDisk`|`string`|Specifies the disk that will be used for the installation process. 
+
+Because the disk discovery order is not guaranteed, the kernel name of the disk can change across booting options for machines with multiple disks. For example, `/dev/sda` becomes `/dev/sdb` and vice versa. To avoid this issue, you must use a persistent disk attribute, such as the disk World Wide Name (WWN), for example: `/dev/disk/by-id/wwn-<disk-id>`.
+
+|====
+
+.Optional specifications
+[options="header"]
+[cols="2a,1a,4a"]
+|====
+|Specification|Type|Description
+|`pullSecret`|`string`|Specifies the pull secret to use during the precache process. The pull secret contains authentication credentials for pulling the release payload images from the container registry.
+|`precacheBestEffort`|`boolean`|Enables best effort mode for image precaching. When you enable best effort mode, the precache process attempts to pull images, but failures will not stop the installation process. Precaching is enabled by default.
```

Comment:
lets remove it as this should be only dev flag

Reply:
Done